src/webportal/src/app/user/fabric/user-profile/storage-list.jsx

```diff
@@ -36,20 +36,16 @@ function normalizePath(path) {
 }
 
 function getStorageServerUri(server) {
-  let blobAddress;
-  let fileAddress;
   const data = server.data;
   switch (server.type) {
     case 'nfs':
       return `nfs://${data.address}/${normalizePath(data.rootPath)}`;
     case 'samba':
       return `smb://${data.address}/${normalizePath(data.rootPath)}`;
     case 'azurefile':
-      fileAddress = `${data.accountName}.<AZURE_STORAGE_DNS_SUFFIX>`;
-      return `smb://${fileAddress}/${normalizePath(data.fileShare)}`;
+      return `StorageAccount: ${data.accountName}; FileShare: ${data.fileShare}; Path: `;
     case 'azureblob':
-      blobAddress = `${data.accountName}.<AZURE_STORAGE_DNS_SUFFIX>`;
-      return `http://${blobAddress}/${normalizePath(data.containerName)}`;
+      return `StorageAccount: ${data.accountName}; Container: ${data.accountName}; Path: `;
```

Comment:
could you make the title bold by returning `<b>Storage Account:</b> ...`?

Reply:
Fixed